examples/hello/test.bats

```diff
@@ -3,6 +3,7 @@ load "${CHTEST_DIR}/common.bash"
 setup () {
```

Comment:
I wonder if the place to substitute in the `ch_tag` line is at the top, maybe even before loading `common.bash`? Then everything else can depend on it being set.

Reply:
Good idea. Addressed in the newest commits.